### DISCUSSION TITLE
pylint E1102: func.* is not callable after upgrading to SQLAlchemy 2.0.0

### DISCUSSION THREAD

**Discussion by ijohn** [sqlalchemy/sqlalchemy#9202] [Usage Questions] [answered]
Hi all,

I have a fully working project using SQLAlchemy 1.4.46 where pylint does not have any complaints and the project runs without error with `SQLALCHEMY_WARN_20=1`.

Upon upgrading to SQLAlchemy 2.0.0, pylint 2.15.10 now complains with `func.count is not callable` and `func.now is not callable`.

`func` is `from sqlalchemy import func`

Anyone knows how to fix this?

Thanks a lot!

**Comment by CaselIT** [comment, marked as answer]
Hi,

this is a duplicate of this https://github.com/sqlalchemy/sqlalchemy/issues/9189
tldr: it's a pylint issue

**Reply by ijohn** [reply]
Ah sorry. I searched and the referred issue didn't come up. Thanks!
